docs/workflows/common-workflow-steps/build-cache/cache-pull.md

```diff
@@ -59,6 +59,14 @@ This step contains some input variable(s). It needs these variable(s) to work. T
 | `$AC_CACHE_LABEL`          | User defined cache label to identify one cache from others. Both [**Cache Push**](/workflows/common-workflow-steps/build-cache/cache-push) and **Cache Pull** steps should have the same value to match. | Required |
 | `$AC_REPOSITORY_DIR`       | Specifies the cloned repository path. This path will be generated after running the [**Git Clone**](/workflows/common-workflow-steps/git-clone) step. | Optional |
 
+:::caution
+
+The build token ID, generated at the start of the build, expires after **3 hours**. If the **Cache Pull** or **Cache Push** step starts beyond this time, caching will fail.
+
+This may lead to a `404 Error` in the **Cache Pull** step during the next build if the **Cache Push** step is not completed successfully.
+
+:::
```

Comment:
@ftmzhrsckldr If we need the same section (_not a single sentence but a meaningful unit as a block_) in different places, then it will be better to make it as a component and reuse it in the required places.

It will make it easy for future updates and will eliminate the possible maintenance issues that can occur when missed.

You can find the relevant samples in the self-hosted documents. @burakberkkeskin can also help you if you need it.

Reply:
@csonuryilmaz Thx. I didn't know we could use components in the documentation. I consulted Berk and updated the code accordingly. Berk approved. FYI.